lib/pages/home_body_content.dart

```diff
@@ -35,18 +35,42 @@ class _HomeBodyContentState extends State<HomeBodyContent> {
       stream: widget.bloc.stream,
       builder: (
         context,
-        snapshot,
+        AsyncSnapshot snapshot,
       ) {
         return snapshot.hasData
-            ? GridView.count(
-                mainAxisSpacing: widget.mainAxisSpacement,
-                childAspectRatio: widget.childAspectRatio,
-                crossAxisCount: widget.crossAxisCount,
-                children: buildList(
-                  context,
-                  snapshot,
-                ),
-              )
+            ? snapshot.data.results.length > 0
```

Comment:
This can be snapshot.data.results.isNotEmpty?

Reply:
Fixed